knowage-vue/src/modules/managers/hierarchyManagement/HierarchyManagementTechnicalTab/HierarchyManagementSourceCard/HierarchyManagementSourceCardDescriptor.json

```diff
@@ -0,0 +1,3 @@
+{
+    "hierarchyTypes": ["MASTER", "TECHNICAL"]
+}
```

Comment:
Seems to be equal to [HierarchyManagementHierarchiesCardDescriptor.json](https://github.com/KnowageLabs/Knowage-Server/pull/486/files#diff-b94c3e41572ace802e24884826388fa9f957cd5c8672599bdfc714324c7aa3c6) 
Is it possible to put them together or they need to be separate for different component nature?